### PR TITLE
Added possibility to deploy env with plugins

### PR DESCRIPTION
**Pull request by AlexGromov**
1) Added possibility to deploy env with detach-database, detach-keystone
and/or detach-rabbitmq plugins. For that we should set the corresponding
flags to TRUE, download plugins and set variables with paths to these
plugins.
2) Edited setting block storage: now we can choose between LVM and CEPH
usage. LVM volume is used by default.
3) Removed FUEL_DEV_VER parameter as it is required only for fuel-qa so it
is better to take it from fuel-qa requirements.
4) Nodes count to prepare is calculated now using the sum of
CONTROLLERS_COUNT, COMPUTES_COUNT, IRONICS_COUNT and
SEPARATE_SERVICES_COUNT.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import fuelweb_test
+from proboscis import register
+
+
+logger = fuelweb_test.logger
```
Comment (gdyuldin) [inline __init__.py]: logger is not used below

File: __init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    groups_list = [
+        {"groups": ["system_test.deploy_env"],
+         "depends": [
+             "system_test.deploy_env("
+             "3_controllers_2compute_neutron_env)"]}
+    ]
+
+    for new_group in groups_list:
+        register(groups=new_group['groups'],
+                 depends_on_groups=new_group['depends'])
```
Comment (gdyuldin) [inline __init__.py]: No new line at end of file (pep8)

File: deploy_env.sh [hunk 3]
```diff
@@ -198,10 +273,6 @@ fi
 
 source ${V_ENV_DIR}/bin/activate
 pip install -U pip
-pip install git+https://github.com/openstack/fuel-devops.git@${FUEL_DEV_VER} --upgrade
```
Comment (gdyuldin) [inline deploy_env.sh]: There is `dos.py` call in code below (but before installing fuel-qa requirements)

File: __init__.py (hunk 2 above)
Comment (AlexGromov) [inline __init__.py]: Done

File: deploy_env.sh (hunk 3 above)
Comment (AlexGromov) [inline deploy_env.sh]: fixed

**Pull request merged**